copy_1.py

```diff
@@ -0,0 +1,70 @@
+from random import randint
+
+Hidden_Pattern=[[' ']*8 for x in range(8)]
+Guess_Pattern=[[' ']*8 for x in range(8)]
+
+let_to_num={'A':0,'B':1, 'C':2,'D':3,'E':4,'F':5,'G':6,'H':7}
+
+def print_board(board):
+    print(' A B C D E F G H')
+    print(' ***************')
+    row_num=1
+    for row in board:
+        print("%d|%s|" % (row_num, "|".join(row)))
+        row_num +=1
+
+def get_ship_location():
+    #Enter the row number between 1 to 8
+    row=input('Please enter a ship row 1-8 ').upper()
+    while row not in '12345678':
+        print("Please enter a valid row ")
+        row=input('Please enter a ship row 1-8 ')
+    #Enter the Ship column from A TO H
+    column=input('Please enter a ship column A-H ').upper()
+    while column not in 'ABCDEFGH':
+        print("Please enter a valid column ")
+        column=input('Please enter a ship column A-H ')
+    return int(row)-1,let_to_num[column]
+
+#Function that creates the ships
+def create_ships(board):
+    for ship in range(5):
+        ship_r, ship_cl=randint(0,7), randint(0,7)
+        while board[ship_r][ship_cl] =='X':
+            ship_r, ship_cl = randint(0, 7), randint(0, 7)
+        board[ship_r][ship_cl] = 'X'
+
+
+
+def count_hit_ships(board):
+    count=0
+    for row in board:
+        for column in row:
+            if column=='X':
+                count+=1
+    return count
+
+create_ships(Hidden_Pattern)
+#print_board(Hidden_Pattern)
+turns = 10
+while turns > 0:
```

Comment:
I'm unsure how seriously we want to take it but if we plan on building upon this game down the road it'll be beneficial, in terms of testing and maintainability, to have our game logic wrapped up inside of a function. i.e.

`def main():

create_ships(Hidden_Pattern)
turns = 10
while turns...


if name == "main":
main()
`